src/main/resources/lang/fr_FR.yml

```diff
@@ -173,6 +195,10 @@ reset:
   msg: Réinitialisation de la pose
   description:
     msg: Réinitialise la position des membrs du corps aux valeurs par défaut
+helpgui:
+  msg: Aide!
```

Comment:
There is a space missing here before `!` (required in French typography). Also, is this `!` needed? I mean, it's there in the original English translation, but it feels a little bit weird.

May I suggest something like this, to keep the exclamation but with better phrasing.

```diff
- Aide!
+ À l'aide !
```